app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -0,0 +1,200 @@
+   * @returns {Function|null}
+   */
+  NewsletterSticky.prototype._scrollDebounceMethod = function(isActive) {
+    return isActive ? utilities.debounce($.proxy(this._handleScroll, this), 100) : null;
+  };
+
+  /**
+   * TODO - write comment
+   */
+  NewsletterSticky.prototype._formAjaxErrorHandler = function() {
+
+    $('.news-signup-sticky__error')
+      .addClass('is-errored news-signup-sticky__error--display');
+
+    $('.news-signup-sticky__text')
+      .removeClass('news-signup-sticky__text--display');
+
+  };
+
+  /**
+   * Make the newsletter module display if scroll is over a certain percentage
+   */
+  NewsletterSticky.prototype._handleScroll = function() {
+    if (this._hasScrolledOverPercentage(this.config.appearsAfterPercentage)) {
+      this.$el.addClass(this.config.visibleClassName);
+      this._setScrollListener(false);
+    }
+    else {
+      return;
+    }
+
+  };
+
+  /**
+   * Works out whether the window has scrolled further than a given percentage
+   * @return {Boolean}
+   */
+  NewsletterSticky.prototype._hasScrolledOverPercentage = function() {
+    var percentScrolled;
+    percentScrolled = this._pageScrolledTo().percentage;
+
+    return (percentScrolled >= this.config.appearsAfterPercentage);
+  };
+
+  /**
+   * Returns the window's scroll top number
+   * @return {Number}
+   */
+  NewsletterSticky.prototype._windowScrollTop = function() {
+    return $(window).scrollTop();
+  };
+
+  /**
+   * Works out the amount of pixels that is needed to be scrolled to before the
+   * newsletter displays
+   * @return {Number}
+   * @deprecated This is no longer used internally, will leave this until production release.
+   */
+  NewsletterSticky.prototype._scrollThreshold = function() {
+    return $(document).height() * (this.config.appearsAfterPercentage / 100);
+  };
+
+  /**
+   * Returns an Object holding the pixels and percentage of the page that has been scrolled.
+   * e.g. return {pixels : x, percentage: n% }
+   * @return {Object}
+   */
+  NewsletterSticky.prototype._pageScrolledTo = function() {
+    var scrollData = {},
+      scrollTop = $(window).scrollTop(),
+      pageHeight = $(document).height(),
+      viewPortHeight = $(window).height(),
+      scrollPercent = (scrollTop / (pageHeight - viewPortHeight)) * 100;
+
+    scrollData.pixels = scrollTop;
+    scrollData.percentage = scrollPercent;
+
+    return scrollData;
+  };
+
+  /**
+   * Handles what happens when the close button is pressed
+   * @param  {Event} e The click event object
+   */
+  NewsletterSticky.prototype._handleCloseClick = function(e) {
+    e.preventDefault();
+
+    if (!$newsletterStickForm.hasClass('news-signup-sticky-form--success')) {
+      MAS.publish('analytics:trigger', {
+        event: 'gaEvent',
+        gaEventCat: 'Newsletter SignUp Sticky',
+        gaEventAct: 'Click',
+        gaEventLab: 'Close button'
+      });
+
+    }
+
+    $.post(newsletterCloseBoxCookieURL,'hello');
```

Comment:
Would we prefer a blank string being sent or do we like to send perhaps a more informed message about the action it's doing?...

'set-close-box-cookie', perhaps?